src/main/webapp/js/components/interface/drawer/TabbedDrawer.js

```diff
@@ -26,7 +26,6 @@ define(function (require) {
             }
             this.openDrawer = this.openDrawer.bind(this);
             this.renderMyLabels = this.renderMyLabels.bind(this);
```

Comment:
No need to bind this either, right?

Reply:
I was using the fat arrow function (doing few tests) since I had a problem with the map scope, I then left the function as it was but I am re-using old declaration and adding the binding here to be in line with the rest of the code. Thx.

Reply:
I have just removed two bindings in the latest commit and everything seems to work fine. Am I missing sth?